Traceback (most recent call last):
  File "MainFile.py", line 111, in <module>
    print(Tuple[0][i].span.strong.a.text, Tuple[1][i].a.text)
IndexError: list index out of range
Traceback (most recent call last):
  File "MainFile.py", line 111, in <module>
    print(Tuple[0][i].span.strong.a.text, Tuple[1][i].a.text)
IndexError: list index out of range
